resources/lib/bossanova808/utilities.py

```
@@ -1,10 +1,12 @@
 import json
 import re
+import xbmc
+import xbmcgui
 import xml.etree.ElementTree as ElementTree
 from urllib.parse import unquote
 
 from bossanova808.constants import *
```

Ruff (0.13.1)

`from bossanova808.constants import *` used; unable to detect undefined names

(F403)